tabs/tab_corr.py

```
@@ -593,9 +798,65 @@ def out_icc_result():
         if result is None:
             return ui.markdown("*Results will appear here after clicking '🔍 Calculate ICC'*")
 
+        res_df = result['results_df']
+        interp_parts = []
+        
+        # ✅ FIX: Iterate through all rows to ensure we capture whatever ICC types are returned
+        # This handles cases where column names might be 'ICC1', 'ICC2', 'ICC(2,1)', etc.
+        for i, row in res_df.iterrows():
```

Ruff (0.14.10)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)